e2e/wdio/core/tests/grid-list.e2e-spec.ts

```diff
@@ -0,0 +1,121 @@
+import { GridListPo } from '../pages/grid-list.po';
+import {
+    click, dragAndDrop, elementArray,
+    getAttributeByName,
+    getCSSPropertyByName,
+    getElementArrayLength, getText,
+    refreshPage,
+    scrollIntoView, waitForClickable, waitForElDisplayed
+} from '../../driver/wdio';
+
+const {
```

Comment:
Please put inside describe block. and you can use gridListPage instead of creating a new object